server/controller/userController.js

```diff
@@ -25,6 +25,29 @@ class Users {
       myRequests,
     });
   }
+
+  /**
+ * @return {Object}
+ * @param {param} req
+ * @param {param} res
+ * @description Returns a request made by a user
+ */
+  static returnRequest(req, res) {
+    const requestId = parseInt(req.params.requestId, 10);
+    const request = requests.find(one => one.id === requestId);
```

Comment:
What's this doing?

Reply:
It is used to find the object with the id that is being sent